src/main/java/sx/blah/discord/handle/impl/events/guild/category/CategoryUpdateEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package sx.blah.discord.handle.impl.events.guild.category;
+
+import sx.blah.discord.handle.obj.ICategory;
+
+/**
+ * Dispatched when a category is updated.
+ *
+ * @see sx.blah.discord.handle.impl.events.guild.channel.ChannelCategoryUpdateEvent
+ */
+public class CategoryUpdateEvent extends CategoryEvent {
+	private final ICategory oldCategory, newCategory;
```

Comment:
Newline after class declaration.
Also each of those fields should be on their own line.